src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml

```diff
@@ -0,0 +1,67 @@
+pattern_name: avoidance_of_food
+pattern_iri: http://purl.obolibrary.org/obo/maxo/patterns/avoidance_of_medical_action.yaml
+description: ""
+
+contributors:
+  - https://orcid.org/0000-0001-7941-2961
+  - https://orcid.org/0000-0002-7356-1779
+  - https://orcid.org/0000-0002-7463-6306
+
+classes:
+  avoidance: MAXO:0001014
+  medical_action: MAXO:0000001
+  food_material: FOODON:00002403
+  
+relations:
+  avoided_medical_action: MAXO:0001015
+  
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym 
+  created_by: oio:created_by
+  date: dc:date
+ 
+vars:
+  date: "'date'"
+  created_by: "'created_by'"
+  medical_action: "'medical_action'"
+  
+name:
+  text: "%s %s avoidance"
+  vars:
+   - food_material
+   - medical_action
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "avoiding %s %s"
+    vars:
+     - food_material
+     - medical_action
+  
+  - annotationProperty: exact_synonym
+    text: "avoid %s %s"
+    vars:
+     - food_material
+     - medical_action
+     
+  - annotationProperty: date
+    text: "%s"
+    vars:
+     - date
+     
+  - annotationProperty: created_by
+    text: "%s"
+    vars:
+     - created_by
+def:
+  text: "A medical requirement of avoiding the %s of %s."
+  vars:
+    - medical_action
+    - food_material
+
+equivalentTo:
+  text: "'avoidance' and ('avoided_medical_action' some ('%s' and has_input some %s))"
```

Comment:
A medical action is anything that alleviates medical problems. How would burger eating (so non-medical actions like consuming unhealthy stuff) fit in the conceptual model here? Maybe we should rethink "avoiding" patterns actually refer to actions in general, not just medical ones.. Avoid smoking etc. Smoking is clearly not a medical action. I think you can go ahead and keep working on this, but my leave the pull request open an maybe put it on the agenda for our next meeting? This should not keep you from working, I think you can go ahead and collect the relevant foodon terms in the TSV.